CHANGES.rst

```diff
@@ -81,6 +81,9 @@ New Features
     currently avaiable mixins. This class does not implement additional
     attributes, methods or a numpy.ndarray-like interface like ``NDDataArray``.
 
+  - ``NDIOMixin`` and subclasses now support reading/writing fits files using
+    the ``read`` or ``write`` method with ``format='simple_fits'``. [#4799]
```

Comment:
Why ``simple_fits`` and not just ``fits``, for consistency with ``Table``? (i.e. can't the more complex readers use ``fits-...``? Note that sub-classes can I think override the meaning of FITS if they want more specific readers.